components/switch/metadata/switch.yml

```diff
@@ -5,7 +5,9 @@ sections:
     description: |
         ### Component renamed
         Toggle is now known as Switch. Replace all `.spectrum-ToggleSwitch*` classnames with `.spectrum-Switch*`.
-
+        ### T-shirt sizing
+        Add the class `spectrum-Switch-sizeM` to specify the t-shirt size.
+        The previous default size is now size M.
```

Comment:
Thank you for adding these t-shirt sizes! 🙌🏻 

Would you mind updating the language here to something like:

`Switch now supports t-shirt sizing and requires that you specify the size by adding a .spectrum-Switch--size* class. The default size is "medium".`